enterprise/api_client/lms.py

```diff
@@ -22,7 +22,10 @@
     from openedx.core.djangoapps.embargo import api as embargo_api
 except ImportError:
     embargo_api = None
-
+try:
```

Comment:
I understand the purpose of this however I think we can added to the previous definitions instead of using a new try/catch block. @alexjmpb

Reply:
I think that every requirement should have it's own try/except block since, otherwise, all requirements would be set to None regardless if it's only one that failed.